app/forms/employment/EmployerNameForm.scala

```diff
@@ -27,15 +27,18 @@ object EmployerNameForm {
 
   val employerName: String = "name"
   val charLimit: Int = 74
+  val regex: String = "^[0-9a-zA-Z\\\\{À-˿’}\\- _&`():.'^]{1,74}$"
```

Comment:
I don't think the input box is accepting commas

<img width="655" alt="Screenshot 2021-08-10 at 10 10 47" src="https://user-images.githubusercontent.com/74660663/128841094-3765cea9-1e61-4274-8286-26d4027f58a8.png">

Reply:
I need to ask content about this